src/mx/Domain.java

```diff
@@ -0,0 +1,25 @@
+package mx;
+
+public class Domain {
```

Comment:
This name is missleading - should be sth like `DomainStatus`, because it contains information about domain url and availability status